src/App.js

```diff
@@ -1,61 +1,85 @@
-import React from "react";
+import React, { useState } from "react";
 import "./App.css";
+import TodoHeader from "./components/TodoHeader";
+import TodoList from "./components/TodoList";
+import TodoFooter from "./components/TodoFooter";
+import { FILTER_TYPE } from "./utils/constants";
 
 function App() {
+  const [todos, setTodos] = useState([
+    { id: 1, title: "새로운 타이틀", done: false, editing: false },
+    { id: 2, title: "완료된 타이틀", done: true, editing: true },
+    { id: 3, title: "완료된 타이틀", done: true, editing: false },
+  ]);
+  const [nextId, setNextId] = useState(4);
+  const [filter, setFilter] = useState(FILTER_TYPE.ALL);
+
+  const onCreate = ({ title, done, editing }) => {
+    const newTodos = todos.concat({
+      id: nextId,
+      title: title,
+      done: done,
+      editing: editing,
```

Comment:
[링크](https://infoscis.github.io/2018/01/25/ecmascript-6-expanded-object-functionality/)를 보시면 객체의 프로퍼티와 로컬 변수의 이름이 같다면 생략할 수 있는 ES6문법이 있습니다! 
```suggestion
      editing,
```
정도로 수정할 수 있겠네요!

Reply:
반영했습니다 👍